tensorflow/training/docker/2.14/py3/Dockerfile.cpu

```diff
@@ -0,0 +1,310 @@
+FROM ubuntu:20.04 AS base_image
+
+ENV DEBIAN_FRONTEND=noninteractive \
+    LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/usr/local/lib"
+
+RUN apt-get update \
```

Comment:
nit: is it worth maintaining this in its own multistage target, as opposed to just merging this statement with the apt-get commands in line 50?

Reply:
We have this as a "header" for all the dockerfiles since we want to perform a blanket OS package upgrade before going into further installation steps. This has helped us figure out some OS package installation issues in the past, but if there is a significant improvement in image size, it might be worth it to merge this step.

I believe we've left it this way because it isn't a significant enough difference in image size, it keeps the dockerfile readable, and we were originally planning on breaking our builds up into modular images.

Until we have a larger review, I'd rather leave this separate multistage target alone. But it is a good item for a retrospective.